### DISCUSSION TITLE
Disabling Collision of Bodies Connected with a Joint

### DISCUSSION THREAD

**Discussion by dafteran4** [avianphysics/avian#259] [Q&A] [answered]
Is there a way to disable collisions between bodies that are connected with a joint? I am reading through docs, I am aware of `CollisionLayers`, but that doesn't seem quite right for the purpose.

**Comment by askor** [comment]
Making a Collision layer for join bodies would be the right tool for this problem

**Reply by ArmoredPony** [reply]
what if one needs to disable collisions between a number of pairs of objects connected with joints?

**Comment by Jondolf** [comment, marked as answer]
Hi, this is now finally implemented with the `JointCollisionDisabled` component added in #799 :)
